care/facility/models/daily_round.py

```diff
@@ -122,7 +122,9 @@ class InsulinIntakeFrequencyType(enum.Enum):
     ]
 
     consultation = models.ForeignKey(
-        PatientConsultation, on_delete=models.PROTECT, related_name="daily_rounds"
+        PatientConsultation,
+        on_delete=models.PROTECT,
+        related_name="daily_rounds",
```

Comment:
are these due to auto formatting, your formatter is not picking up the correct line length

Reply:
I am running pre-commit. All the files are formatted before I push the changes